### DISCUSSION TITLE
Is there a place to change keybinds?

### DISCUSSION THREAD

**Discussion by thespyboss** [xournalpp/xournalpp#3815] [Q&A]
I am trying to find out what the keybind for the experimental toolbox is and can't find it anywhere. I want to map it to a button on my Wacom Intuos but I just don't know how to.

**Comment by Technius** [comment]
To bind the button on stylus only, use the configuration options in `Preferences > Stylus > Stylus Buttons`.

I don't think there is a built-in keyboard button binding for the floating toolbox right now (correct me if I'm wrong), but a binding can be created through a plugin.
